src/qt/guiutil.cpp

```diff
@@ -399,7 +400,13 @@ bool openBitcoinConf()
     configFile.close();
 
     /* Open bitcoin.conf with the associated application */
-    return QDesktopServices::openUrl(QUrl::fromLocalFile(boostPathToQString(pathConfig)));
+    bool res = QDesktopServices::openUrl(QUrl::fromLocalFile(boostPathToQString(pathConfig)));
+#if defined(Q_OS_MAC)
+    if (!res)
+        res = QProcess::startDetached("/usr/bin/open", QStringList() << "-t" << boostPathToQString(pathConfig));
```

Comment:
nit: could `if` statement be placed in braces?

micronit: using an initializer list is shorter: `QStringList{"-t", boostPathToQString(pathConfig)}`

Reply:
fixed in the new commit